src/pybind/mgr/dashboard/frontend/src/app/shared/directives/form-input-disable.directive.ts

```diff
@@ -0,0 +1,27 @@
+import { AfterViewInit, Directive, ElementRef, Optional } from '@angular/core';
+
+import { Permissions } from '../models/permissions';
+import { AuthStorageService } from '../services/auth-storage.service';
+import { FormScopeDirective } from './form-scope.directive';
+
+@Directive({
+  selector:
+    'input:not([cdNoFormInputDisable]), select:not([cdNoFormInputDisable]), [cdFormInputDisable]'
+})
+export class FormInputDisableDirective implements AfterViewInit {
+  permissions: Permissions;
+
+  constructor(
+    @Optional() private formName: FormScopeDirective,
```

Comment:
```suggestion
    @Optional() private formScope: FormScopeDirective,
```

Reply:
Thought I took care of it previously. But done now.